arch/arm/CMakeLists.txt

```diff
@@ -1,26 +1,7 @@
+#
```

Comment:
Not sure you need copyright header here, it's not added in arch cmakelists files. suggest to remove